nextplace/validator/nextplace_validator.py

```diff
@@ -96,6 +98,13 @@ def check_timer_set_weights(self) -> None:
             finally:
                 self.database_manager.lock.release()
 
+    def process_and_send_predictions(self) -> None:
```

Comment:
We should run this in a separate thread. We need to access the database to send the data, and we don't want to main thread to indefinitely block. We need to access the database lock to pull this data from the database.